pycbc/distributions/sky_location.py

```diff
@@ -46,6 +46,86 @@ class UniformSky(angular.UniformSolidAngle):
     _default_azimuthal_angle = 'ra'
 
 
+class UniformCircleSky:
+
+    name = 'uniform_circle_sky'
+    _params = ['ra', 'dec']
+    
+    def __init__(self, **params):
+        self.mean_ra = angle_as_radians(params['mean_ra'])
+        self.mean_dec = angle_as_radians(params['mean_dec'])
+        self.sigma = angle_as_radians(params['sigma'])
```

Comment:
I would rename sigma to radius in this class. Sigma is reminescent of a standard deviation, but in the case of a uniform disk, it makes more sense to talk about radius.